PGIP/Output/Proof.hs

```diff
@@ -30,8 +32,8 @@ import Numeric
 
 import Text.XML.Light (ppTopElement)
 
-type ProofResult = (String, String, String,
-                -- ^(goalName, goalResult, goalDetails
+type ProofResult = (String, String, String, ProverOrConsChecker,
+                -- ^(goalName, goalResult, goalDetails, prover
```

Comment:
there are wrong haddock markers (check using "make doc"). I had to fix them in https://github.com/spechub/Hets/tree/ignazio1977-master